tests/lib/rules/no-multi-comp.js

```diff
@@ -89,6 +89,21 @@ ruleTester.run('no-multi-comp', rule, {
     options: [{
       ignoreStateless: true
     }]
+  }, {
+    // multiple non-components
+    code: [
+      'import React, { createElement } from "react"',
+      'const helperFoo = () => {',
+      '  return true;',
+      '};',
+      'function helperBar() {',
+      '  return false;',
+      '};',
+      'function RealComponent() {',
+      '  return createElement("img");',
+      '};'
+    ].join('\n'),
+    parser: 'babel-eslint'
```

Comment:
I used `babel-eslint` over `sourceType: 'module'` because it was way more popular in the codebase for handling import statements. (git grep sourceType)

Do you think it would be better to apply the module sourceType to the entire test file, just the single test using object.assign, or using `const createElement = require('react').createElement;` in the actual test? 

Maybe a chore issue to clean out all of babel-eslint and replace it with sourceType module. eslint's default parser could handle a vast majority of the tests.